### PR TITLE
feat: add square size field to data root tuple

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

As per the decision in https://github.com/celestiaorg/quantum-gravity-bridge/issues/142, we need to add the block size to the data root tuple.

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: src/DataRootTuple.sol [hunk 1]
```diff
@@ -11,4 +11,6 @@ struct DataRootTuple {
     uint256 height;
     // Data root.
     bytes32 dataRoot;
+    // Celestia original block size.
+    uint256 blockSize;
```
Comment (evan-forbes) [inline src/DataRootTuple.sol:15]: square size or block size? can we get more docs on the units or what this is?

Comment (SweeXordious) [reply src/DataRootTuple.sol:15]: good catch, will update to square size

Comment (SweeXordious) [reply src/DataRootTuple.sol]: [07f464e](https://github.com/celestiaorg/quantum-gravity-bridge/pull/145/commits/07f464e7196198999999c2e12ff78247a78ba36b)

File: src/test/QuantumGravityBridge.t.sol [hunk 2]
```diff
@@ -110,15 +111,16 @@ contract RelayerTest is DSTest {
...
         // https://github.com/celestiaorg/celestia-core/blob/60310e7aa554bb76b735a010847a6613bcfe06e8/crypto/merkle/proof.go#L33-L48
-        bytes32 newTupleRoot = 0x82dc1607d84557d3579ce602a45f5872e821c36dbda7ec926dfa17ebc8d5c013;
+        bytes32 newTupleRoot = 0x3db8fb64a10316e447733af826165d0543b94b9de3b4146dcad70611f5089018;
         // a data root committed to by the above tuple root.
-        bytes32 newTuple = 0x0101010101010101010101010101010101010101010101010101010101010101;
+        bytes32 newTuple = 0x0202020202020202020202020202020202020202020202020202020202020202;
         // the height of the data root.
-        uint256 height = 1;
+        uint256 height = 2;
+        uint256 size = 32;
```
Comment (rootulp) [inline src/test/QuantumGravityBridge.t.sol:119]: [nit] size => squareSize per https://github.com/celestiaorg/celestia-app/issues/687

Comment (SweeXordious) [reply src/test/QuantumGravityBridge.t.sol]: [5a322c9](https://github.com/celestiaorg/quantum-gravity-bridge/pull/145/commits/5a322c97158e41074aeffb793a99adc368b4af0b)

**Review by evan-forbes — APPROVED** [review]

**Review by rootulp — APPROVED** [review]

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**